DXRMapFixups/DeusEx/Classes/DXRFixupM06.uc

```diff
@@ -104,6 +104,18 @@ function PreFirstEntryMapFixes()
                     break;
                 }
             }
+
+            foreach AllActors(class'Button1', b) {
+                if (b.tag == 'Weapons_Lock_broken' || b.tag == 'Missile_door_toggle' || b.tag == 'Weapons_lock') {
```

Comment:
Both the munitions bay buttons have the Event "missile_door" (but only the one on Jock's side has a unique tag), so you can search for the event on those buttons instead